jekyll/_cci2/deployment-examples.md

```diff
@@ -0,0 +1,685 @@
+---
+layout: classic-docs
+title: "Deployment Examples"
+short-title: "Deployment Examples"
+---
+This document presents example config for a variately of popular deployment targets. Many of the examples use orbs: CircleCI and partners have developed a catalogue of orbs that enable you to quickly deploy applications with minimal config. Details of all orbs can be found in the [CircleCI Orbs Registry](https://circleci.com/orbs/registry/).
+
```

Comment:
Are there any orbs for deploying Helm packages to Kubernetes? I feel like that'd be a really great example to have.

Reply:
Oh great, thanks for this idea, I am going to add it to my to do list :-)